eva_submission/eload_preparation.py

```diff
@@ -190,13 +193,13 @@ def find_genome(self):
 
 @retry(tries=4, delay=2, backoff=1.2, jitter=(1, 3))
 def contig_alias_put_db(contig_alias_payload, contig_alias_url, contig_alias_user, contig_alias_pass):
-    for assembly in contig_alias_payload:
-        response_get = requests.get(os.path.join(contig_alias_url, assembly),
-                                    auth=(contig_alias_user, contig_alias_pass))
+    request_url = os.path.join(contig_alias_url, 'v1/admin/assemblies')
 
-        response_get_data = response_get.json()
-
-        if response_get_data.get('page', {}).get('size', 0) != 0:
-            response_put = requests.put(os.path.join(contig_alias_url, assembly),
-                                        auth=(contig_alias_user, contig_alias_pass), json=assembly)
-            response_put.raise_for_status()
+    for assembly in contig_alias_payload:
+        response = requests.put(os.path.join(request_url, assembly), auth=(contig_alias_user, contig_alias_pass))
+        if response.status_code == 200:
+            logger.info(f'Assembly accession {assembly} successfully added to Contig-Alias DB')
+        elif response.status_code == 409:
+            logger.warning(f'assembly already exist in Contig-Alias DB. Response: {response.text}')
+        else:
+            logger.error(f'Could not save assembly to Contig-Alias DB. Error : {response.text}')
```

Comment:
We should discuss whether we still want submission processing to crash in this case, as the previous implementation did...

Reply:
I'd also include the assembly accession in these log messages, as you did in the other case.

Reply:
done